demo.js

```diff
@@ -18,3 +18,5 @@ const run = async () => {
 };
 
 run();
+
+console.log(JSON.stringify(parseHTML(getDemoData()), null, 2));
```

Comment:
Maybe remove this to keep the demo from getting cluttered with dev lines.

Reply:
Done!